soql-server-pg/src/main/scala/com/socrata/pg/server/analyzer2/ProcessQuery.scala

```diff
@@ -410,10 +420,10 @@ object ProcessQuery {
         writer.write("\"schema\":")
 
         @AutomaticJsonEncode
-        case class Field(c: ColumnName, t: SoQLType)
+        case class Field(c: ColumnName, t: SoQLType, h: Option[JValue])
 
-        val reformattedSchema = nameAnalysis.statement.schema.values.map { case Statement.SchemaEntry(cn, typ, _isSynthetic) =>
-          Field(cn, typ)
+        val reformattedSchema = nameAnalysis.statement.schema.values.map { case Statement.SchemaEntry(cn, typ, _hint, _isSynthetic) =>
+          Field(cn, typ, hintsByName.get(cn))
```

Comment:
Out of curiosity, what's the benefit of pulling the hints out into hintsByName and then accessing them that way, rather than using the parameter here? Is it just for ETag stuff?

Reply:
It's because the analysis is post-rollups, which may/will have destroyed any hints.

Reply:
Added a comment to that effect here.